### PR TITLE
[#14] DKAN script is not really fit for purpose

### PR DESCRIPTION
**Pull request by caprenter**
I've had a go at making this script better, but I'm struggling
I suspect someone can do a better job either from scratch or by tweaking this.

### CODE REVIEW

**Comment by caprenter** [comment]
I've assigned this to you Ben. I think you might be able to look at this quickly and improve it, or pass it to someone else to improve.

I can see it being a useful tool in  the coming weeks, so a quick fix now would be good.

**Comment by Bjwebb** [comment]
@caprenter Taken a brief look, and I think I should be able to get something committed on Monday.

A few questions though:
- What's the purpose of the last modified checking? As far as I can tell you're downloading all the data every time anyway. Did you compare how long the script took before/after that change?
- Has the last modified checking ever worked correctly? You seem to be comparing strings with English names in, which won't give you the correct answer. Also, you very rarely want the time module (it does things like returning you strings), and generally want to use the datetime module instead.
- Did this ever correctly handle datasets with multiple resources (like http://data.threesixtygiving.org/dataset/oxfordshire-community-foundation) - are you happy for me to switch to a naming scheme other than package name to deal with these (or is this a problem that should be corrected on the registry)?

**Comment by caprenter** [comment]
The answers are - I tried to add something that means that if a remote dataset has not changed, then don't pull it again. I think it worked, but you will know better.

Why?
Seems to be sensible not to bother getting it again if it has not changed
If nothing much has changed the script should run faster
Some of the data is 'quite big' so shy sit around waiting for it to be re-downloaded
Get some idea of what has changed and what has not

Right now, a script that works would be good.

Originally, this just pulled CSVs, and there wasn't much data so I wasn't particularly worried about multiple datasets.

So basically, you have free reign here

**Comment by Bjwebb** [comment]
I've commited and suggested my changes at https://github.com/ThreeSixtyGiving/data-conversion/pull/16

Suspect this pull request can now be closed, but assigning to you to make the final call.
